modules/@ergonode/ui/src/components/ParagraphWithLinks/ParagraphWithLinks.vue

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright © Bold Brand Commerce Sp. z o.o. All rights reserved.
+ * See LICENSE for license details.
+ */
+<script>
+import {
+    isEmpty,
+} from '@Core/models/objectWrapper';
+
+export default {
+    name: 'ParagraphWithLinks',
+    props: {
+        paragraph: {
+            type: String,
+            required: true,
+        },
+        links: {
+            type: Object,
+            default: () => ({}),
+        },
+    },
+    render(createElement) {
```

Comment:
We do not use render function at all - expect renderless components, I would stick to one approach